.github/workflows/coverity.yml

```diff
@@ -50,32 +50,17 @@ jobs:
           $GCOV --version
           meson --version
           ninja --version
-      - name: Install Coverity
-        shell: bash
-        run: |
-          .github/installCoverity.sh
-        env:
-          COVERITY_SCAN_TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}
-          COVERITY_PROJECT_NAME: substrate
-      - name: Run Coverity
+      - name: Configure
         shell: bash
         run: |
+          echo "SUBSTRATE_VERSION=`meson introspect build --projectinfo | jq -r '.["version"]'`-`git rev-parse --short HEAD`" >> $GITHUB_ENV
```

Comment:
This line needs moving one down as we talked about and discovered in trying a Coverity run, and the branch rebased - once that's been done we'll get this merged.